fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -59,7 +39,7 @@ export default Ember.Route.extend(NeedsExistingManifestHelpers, {
   },
 
   setupController(controller, modelHash) {
-    const model = modelHash.reviewModel;
+    const model = this.modelFor('review');
```

Comment:
`modelHash.reviewModel` also caused an error that was hard to debug. It seems more straight forward just to assign it to `this.modelFor('review')`